ext/geometryset.jl

```diff
@@ -52,14 +52,20 @@ function vizgset!(plot, ::Type{<:𝔼}, ::Val{0}, ::Val, geoms::ObservableVector
 end
 
 function vizgset!(plot, ::Type{<:𝔼}, ::Val{1}, ::Val, geoms, colorant)
+  showpoints = plot[:showpoints]
+
   meshes = Makie.@lift discretize.($geoms)
   vizmany!(plot, meshes, colorant)
-  showfacets1D!(plot, geoms)
+
+  if showpoints[]
```

Comment:
are you positive that adding this new conditional retains the same behavior as before?